cirq-core/cirq/contrib/acquaintance/executor.py

```diff
@@ -29,9 +30,6 @@
 )
 from cirq.contrib.acquaintance.mutation_utils import expose_acquaintance_gates
 
-if TYPE_CHECKING:
-    import cirq
-
```

Comment:
`import cirq` should only be done for type checking. So revert this change and remove the `import cirq` statement on top.